enable/savage/trait_defs/ui/qt4/svg_editor.py

```diff
@@ -17,12 +17,14 @@
 
 from io import BytesIO
 from xml.etree.cElementTree import ElementTree
+from PySide6.QtSvgWidgets import QSvgWidget
```

Comment:
This import will only succeed if PySide6 is installed. What happens if someone wants to use this code with PyQt6 instead of PySide6?